src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -62,7 +71,7 @@ internal static string GetPipePath(string serverName, string pipeName)
 
         /// <summary>Throws an exception if the supplied handle does not represent a valid pipe.</summary>
         /// <param name="safePipeHandle">The handle to validate.</param>
-        internal static void ValidateHandleIsPipe(SafePipeHandle safePipeHandle)
+        internal void ValidateHandleIsPipe(SafePipeHandle safePipeHandle)
```

Comment:
Right. Is there some reason I've missed that this shouldn't be an instance method?